{
    "affected": [
        {
            "package": {
                "ecosystem": "Photon OS:4.0",
                "name": "python3-werkzeug",
                "purl": "pkg:rpm/vmware/python3-werkzeug?distro=photon-4"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "0"
                        },
                        {
                            "fixed": "2.2.2-2.ph4"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        }
    ],
    "id": "PHSA-2025-4.0-0788",
    "modified": "2025-04-26T05:29:33Z",
    "published": "2025-04-24T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-4.0-788"
        }
    ],
    "related": [
        "CVE-2023-25577",
        "CVE-2023-23934"
    ]
}